scene/2d/remote_transform_2d.cpp

```diff
@@ -63,7 +63,21 @@ void RemoteTransform2D::_update_remote() {
 		return;
 
 	//todo make faster
-	n->set_global_transform(get_global_transform());
+	if (use_global_coordinates) {
+		if (update_remote_position)
+			n->set_global_position(get_global_position());
+		if (update_remote_rotation)
+			n->set_global_rotation(get_global_rotation());
+		if (update_remote_scale)
+			n->set_global_scale(get_global_scale() + Vector2(1, 1));
```

Comment:
I added (1, 1) to scale because I found that nodes had their scale set to zero (or rather 0.00001)  if the remote transform had a scale of (1, 1) and adding one fixed it. Probably a bug in my code.